src/host.rs

```diff
@@ -16,7 +16,7 @@ pub struct MiralisContext {
     /// Hardware capabilities of the core (hart).
     pub hw: HardwareCapability,
     /// List of devices with PMP
-    pub devices: [device::VirtDevice; 1],
+    pub devices: [device::VirtDevice; 2],
```

Comment:
@CharlyCst Do we want to have a test device on each platform?

Reply:
Let's say yes for now, I don't expect it to add much overhead in terms of space or performance.
In the future we will need to have a better way of describing platforms and their devices, but we don't have enough devices to worry about it yet :)